kaizen-openapi-parser/src/main/java/com/reprezen/kaizen/oasparser/jsonoverlay/JsonLoader.java

```diff
@@ -40,8 +41,10 @@ public JsonNode load(URL url) throws IOException {
 		if (cache.containsKey(urlString)) {
 			return cache.get(urlString);
 		}
-		String json = IOUtils.toString(url.openStream(), Charsets.UTF_8);
-		return loadString(url, json);
+		try(InputStream in = url.openStream()) {
+			String json = IOUtils.toString(in, Charsets.UTF_8);
```

Comment:
@maybeec Thanks, this looks perfect - and thanks for the second "reformatting" push to make the diff more comprehensible!

I expect to create a new release this week, and will include this.

Reply:
Great to hear! Thanks